lib/driver.js

```diff
@@ -18,6 +18,7 @@ import { setLocale, setPreferences } from './settings';
 import { runSimulatorReset, isolateSimulatorDevice, checkSimulatorAvailable,
          moveBuiltInApp, getAdjustedDeviceName, endSimulator, runRealDeviceReset } from './device';
 import { version } from '../../package.json'; // eslint-disable-line import/no-unresolved
+import { IWDP } from './iwdp';
```

Comment:
I just pushed a commit that exports the IWDP.

The next step, once this PR is closed, will be testing it in xcuitest-driver